Mathlib/MeasureTheory/Integral/Bochner.lean

```diff
@@ -1563,6 +1563,20 @@ theorem integral_tsum {ι} [Countable ι] {f : ι → α → G} (hf : ∀ i, AES
     exact hx.of_norm.hasSum
 #align measure_theory.integral_tsum MeasureTheory.integral_tsum
 
+lemma hasSum_integral_of_summable_integral_norm {ι} [Countable ι] (F : ι → α → E)
```

Comment:
could you also write a lemma `integral_tsum'` under your assumptions? (it is a direct consequence of the one you've just written)